requirements.txt

```diff
@@ -8,6 +8,6 @@ scipy>=1.4
 spacy==2.2.4
 tensorboard>=2.1.0
 tensorflow>=2.2.0rc1
-tokenizers>=0.7
+tokenizers==0.7
 tqdm==4.45.0
-transformers>=2.9.0
+transformers==2.10.0
```

Comment:
is there any reason of why we don't want the latest `transformers` and `tokenizers`?

Reply:
I get this error using latest transformers and tokenizers
<img width="1177" alt="Screen Shot 2020-07-13 at 2 50 08 PM" src="https://user-images.githubusercontent.com/64120158/87341774-28a4d700-c518-11ea-89ba-1090a600bdc0.png">